draft-ietf-openpgp-pqc.md

```diff
@@ -863,9 +861,9 @@ This group of algorithms is henceforth referred to as "SLH-DSA-SHAKE code points
 {: title="SLH-DSA-SHAKE algorithm code points and the corresponding artifact lengths in octets." #slhdsa-artifact-lengths}
 Algorithm ID reference   |  SLH-DSA-SHAKE public key | SLH-DSA-SHAKE secret key | SLH-DSA-SHAKE signature
 ----------------------:  |  ------------------ | ------------------ | ------------------
-TBD (SLH-DSA-SHAKE-128s) |  32                 | 64                 | 7856
```

Comment:
The column says "Algorithm ID reference", not "Algorithm ID". So unless the column label is changed, I would leave the textual algorithm label there.  The ID itself could maybe be added in a new column (leftmost)?

The same applies to the changes to the next table about the hash binding.

Reply:
It's more reliable for now to keep the textual algorithm description, I think. The temporary inofficial IDs are not a reliable identification token. They might change and then readers might be confused when they see the next version.

Reply:
I can also live with the proposed change, but I think for now being more explicit in those tables might be better.